cfgov/unprocessed/css/molecules/image-text-50-50.less

```diff
@@ -82,7 +82,6 @@
 */
 
 .m-image-text-50-50 {
-    padding-bottom: unit( @grid_gutter-width * 2 / @base-font-size-px, em );
```

Comment:
Fixed it by setting spacing after each `.m-image-text-50-50` and removing the block spacing on the bottom around the group.